### PR TITLE
osx-arm64 target

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, rerender.

**Comment by isuruf** [comment]
Merged to `rc` branch.

File: recipe/patches/0002-run-ldid-after-install_name_tool.patch [hunk 1]
```diff
@@ -1,57 +1,76 @@
...
-+        memcpy(ldidcommand, "ldid -S ", 8);
-+        memcpy(ldidcommand + 8, input, strlen(output) + 1);
-+        system(ldidcommand);
-+    }
++	if (isarm64 == 1) { /* cctools-port */
++	    bufsize = MAXPATHLEN;
++	    p = buf;
++	    i = _NSGetExecutablePath(p, &bufsize);
++	    if(i == -1){
++		p = allocate(bufsize);
```
Comment (beckermr) [inline recipe/patches/0002-run-ldid-after-install_name_tool.patch]: is the memory attached to p being leaked here?

Comment (isuruf) [reply recipe/patches/0002-run-ldid-after-install_name_tool.patch]: yes. it's a common code pattern used throughout the cctools codebase.

Comment (beckermr) [reply recipe/patches/0002-run-ldid-after-install_name_tool.patch]: kk - I guess these tools return quickly so no harm

**Pull request merged**